docker-compose.gundb.yml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  gun:
+    image: skynetlabs/gundb-relay
+    container_name: gundb-relay
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment: 
+      - PORTAL=http://${SERVER_DOMAIN}
```

Comment:
we should probably use skyd or nginx local ip directly here, otherwise it's making unnecessary roundtrip

also we'll need to add support for user authentication